routes/api/users.js

```diff
@@ -23,4 +24,8 @@ router.post("/users", (req, res, next) => {
     }).catch(next);
 });
 
+// Reset password 
+router.post('/users/reset', userController.resetPassword);
```

Comment:
@fridolinho , for the sake of having the common naming if the routes. Routes that are related to user(signup, login, reset password and update password) we can use `{{host}}/api/v1/{{name}}`.